app/models/licence_transaction.rb

```diff
@@ -17,6 +17,10 @@ def taxons
     []
   end
 
+  def change_note_required?
+    false
```

Comment:
Sure, have added a change note 👍 

> Could a default change note be set instead? i.e. something like "Imported from Publisher"?

Hmm, change notes are meant to represent a public facing change, rather than documenting an internal migration so I was hesitant to to add that sort of note, however it doesn't matter at this stage so I have added that change note... for now and we can discuss further at a later date 😄 

> I know that change notes are not being displayed on licences at the moment, but it could be useful to keep them for email alerts.

We don't currently support email alerts for licences unless we choose to enable them on the finder but fair point.